packages/sanity/src/core/store/_legacy/history/useTimelineSelector.ts

```diff
@@ -9,9 +9,14 @@ import {type TimelineState, type TimelineStore} from './useTimelineStore'
  * @internal
  */
 export function useTimelineSelector<ReturnValue>(
-  timelineStore: TimelineStore,
+  timelineStore: TimelineStore | undefined,
```

Comment:
A tad concerned about this change, this will be a breaking change since users so far haven't had to be concerned about potentially sending an undefined through here.

Though the more I think about it might be fine? Since overall the users might need to make some extra changes? We do still keep the legacy components 🤔